elements/rh-secondary-nav/demo/demo.css

```diff
@@ -23,3 +32,12 @@ rh-secondary-nav-menu#custom-grid::part(sections) {
     --rh-secondary-nav-logo-max-width: 13.4em;
   }
 }
+
+/* D7 font-size override for lightdom
+rh-secondary-nav > [slot="logo"] {
+  font-size: 1.8rem !important;
+}
```

Comment:
Can we do this with CSS vars?  I think we should only keep the `!important` property internal to the component.

Reply:
Kinda like:

```
<style>
  :root {
    --rh-font-size-text-xl: 1.125em;
    --rh-line-height-body-text: 150%;
    --rh-font-size-body-text-xs: 0.75em;
    --rh-font-size-body-text-sm: 0.875em;
    --rh-font-size-code-xs: 0.75em;
    --rh-footer-link-header-mobile-font-size: 0.875em;
  }
  rh-footer::part(links-accordion-header) {
    --rh-footer-link-header-font-size: 0.875em;
  }
</style>
```

Reply:
Yeah nice idea. It can be a "private" underscore name